### ISSUE TITLE
Featured image block: Border width and border radius not applied on front end.

### ISSUE THREAD

**Issue by kebbet** [WordPress/gutenberg#45142]
### Description

When defining a border (width and color) and/or border-radius on a ”Featured image” block, and also restricting its height, the border and border-radius options are not applied on front end.

### Step-by-step reproduction instructions

1. Insert featured image block
2. Restrict height to 100px in Diemension section.
3. Add border (color + width) in Border section
4. and/or add border-radius in Border section.
5. See the options applied in editor.
6. See the options missing on front end.

### Screenshots, screen recording, code snippet

Editor
<img width="578" alt="bild" src="https://user-images.githubusercontent.com/11491369/196894177-11a63a3a-6452-486a-8f79-c91dbd1c56e2.png">

Front end.
<img width="499" alt="bild" src="https://user-images.githubusercontent.com/11491369/196894215-7231941f-c9ab-4dd7-b715-00f2eab601c2.png">

### Environment info

- WordPress trunk, Twenty-Twenty-Three and Twenty-Twenty-Two.
- 

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by ntsekouras** [comment]
👋 @kebbet thanks for reporting this! I couldn't reproduce the issue though. --cc @aaronrobertshaw

**Comment by kebbet** [comment]
@ntsekouras hmmm. Even with the gutenberg plugin inactive and just WordPress trunk?

**Comment by aaronrobertshaw** [comment]
Thank you for flagging this @kebbet 👍 

With Gutenberg activated, I couldn't replicate it. However, I can with it deactivated. 

This issue was actually fixed in https://github.com/WordPress/gutenberg/pull/44213, I've added a label to backport it to the WP 6.1 RC.

The application of block support inline styles was also broken when the overlay feature was added to the Post Featured Image block. That fix (https://github.com/WordPress/gutenberg/pull/44286) however has already been backported.

Are you happy to close this issue pending backport of #44213?
